client/app/lib/accessibility.ts

```diff
@@ -0,0 +1,21 @@
+export function srNotify(text: string, expiry = 1000, container = document.body): void {
+  const element = document.createElement("div");
+  const id = `speak-${Date.now()}`;
+
+  Object.assign(element, {
+    id,
+    className: "sr-only",
+    textContent: text,
+  });
+
+  [
+    ["role", "alert"],
+    ["aria-live", "assertive"],
+  ].forEach(([attr, value]) => element.setAttribute(attr, value));
+
+  container.appendChild(element);
+
+  setTimeout(() => {
+    container.removeChild(element);
```

Comment:
May need to check that `container` still exists (if I remove the container, and this fires, could get ugly).

Reply:
Container will stay valid while there is at least one reference to it. It means - even if it will be removed from DOM - here it will stay valid, and nothing bad will happen (at least until we decide to access its parent node). However, bad things may happen if we'll decide to remove a child that already does not belong to the element. So this `removeChild` call needs to be wrapped with `try..catch`